custom_components/tahoma/cover.py

```diff
@@ -131,10 +134,6 @@ async def async_set_cover_position(self, **kwargs):
         """Move the cover to a specific position."""
         position = 100 - kwargs.get(ATTR_POSITION, 0)
 
-        # HorizontalAwning devices need a reversed position that can not be obtained via the API
-        if "Horizontal" in self.device.widget:
-            position = kwargs.get(ATTR_POSITION, 0)
-
```

Comment:
Why remove this part?

Reply:
Based on https://github.com/iMicknl/ha-tahoma/issues/103#issuecomment-663761420.

Reply:
Hum, oki. I should try also with own Awning.

Reply:
@tetienne do you have a Horizontal Awning as well? I am not sure about this change, hence I made it a draft and looking for testers.

Reply:
Yes I have this one: io:HorizontalAwningIOComponent. I can test at noon.

Reply:
Just test it, and the behavior is not good. The previous one was ok.

Reply:
@tetienne thanks for confirming. This brings me back to the drawing table. Weird that some devices have a totally different behavior.